### PR TITLE
Fix 908974 and 871079

### PR DESCRIPTION
**Pull request by jsalant22**
Check for projects with 0 designs, before trying to access any designs (fixes 908974)

Check for valid interaction after a domain is run. Could be invalid due to disabled Radio pairs. (fixes 871079)

### CODE REVIEW

**Comment by jsalant22** [comment]
@ansys-reviewer-bot review

File: examples/07-EMIT/interference_gui.py [hunk 1]
```diff
@@ -57,7 +57,7 @@ def install(package):
 
 # Add emitapi to system path
 emit_path = os.path.join(desktop.install_path, "Delcross")
```
[the remark this answers is not in the document]
Comment (jsalant22) [reply examples/07-EMIT/interference_gui.py:59]: Suggested change makes no sense and changes functionality.

File: examples/07-EMIT/interference_gui.py [hunk 2]
```diff
@@ -244,8 +244,20 @@ def open_file_dialog(self):
             # Close previous project and open specified one
             if self.emitapp is not None:
                 self.emitapp.close_project()
```
[the remark this answers is not in the document]
Comment (jsalant22) [reply examples/07-EMIT/interference_gui.py:246]: This line is necessary and used to close projects when loading a new project.

File: examples/07-EMIT/interference_gui.py [hunk 3]
```diff
@@ -244,8 +244,20 @@ def open_file_dialog(self):
...
+                self.emitapp = None
             desktop_proj = desktop.load_project(self.file_path_box.text())
 
+            # check for an empty project (i.e. no designs)
+            if isinstance(desktop_proj, bool):
+                self.file_path_box.setText("")
```
[the remark this answers is not in the document]
Comment (jsalant22) [reply examples/07-EMIT/interference_gui.py:252]: We're closing the _current_ project, then checking to see that the new project actually has designs (i.e. not empty)

File: pyaedt/emit_core/results/revision.py [hunk 4]
```diff
@@ -477,6 +477,10 @@ def interference_type_classification(self, domain, use_filter=False, filter_list
                         domain.set_receiver(rx_radio, rx_band)
                         domain.set_interferer(tx_radio, tx_band)
                         interaction = self.run(domain)
+                        # check for valid interaction, this would catch any disabled radio pairs
+                        if not interaction.is_valid():
+                            continue
```
[the remark this answers is not in the document]
Comment (jsalant22) [reply pyaedt/emit_core/results/revision.py:482]: Indentation changes would break script... comment seems pretty descriptive.

File: pyaedt/emit_core/results/revision.py [hunk 5]
```diff
@@ -622,6 +626,9 @@ def protection_level_classification(
                     domain.set_receiver(rx_radio, rx_band)
                     domain.set_interferer(tx_radio, tx_band)
                     interaction = self.run(domain)
```
[the remark this answers is not in the document]
Comment (jsalant22) [reply pyaedt/emit_core/results/revision.py:628]: Indentation changes would break script... comment seems pretty descriptive.

File: pyaedt/emit_core/results/revision.py [hunk 6]
```diff
@@ -648,7 +655,7 @@ def protection_level_classification(
                         else:
```
[the remark this answers is not in the document]
Comment (jsalant22) [reply pyaedt/emit_core/results/revision.py:655]: N/A

File: examples/07-EMIT/interference_gui.py [hunk 7]
```diff
@@ -566,8 +578,11 @@ def closeEvent(self, event):
         msg.setWindowTitle("Closing GUI")
         msg.setText("Closing AEDT, please wait for GUI to close on its own.")
```
Comment (PipKat) [inline examples/07-EMIT/interference_gui.py:579]: We should omit "please" from the message. This word isn't necessary in technical documentation. I'd suggest rewording the message to "Closing AEDT. Wait for the GUI to close on its own."

File: pyaedt/emit_core/results/revision.py [hunk 8]
```diff
@@ -648,7 +655,7 @@ def protection_level_classification(
...
                             filtering = True
 
-                        if instance.get_value(mode_power) > max_power and filtering:
+                        if instance.has_valid_values() and instance.get_value(mode_power) > max_power and filtering:
```
Comment (myoung301) [inline pyaedt/emit_core/results/revision.py:658]: @jsalant22 Are there some reasons for not having valid values that could indicate an interference problem (e.g. saturation)? If so, should we be treating those as one of the problematic classifications? Right now, if I'm following the code correctly, the max_power value will be -200 for any case where `instance.has_valid_values()=false` and it will be treated like there is no interference issue.

**Review by myoung301 — APPROVED** [review]
